__tests__/utils.test.js

```diff
@@ -1,104 +1,316 @@
+const { commentData } = require("../db/data/test-data");
```

Comment:
I think this is just Prettier reformatting here, but you shouldn't have to change this file